src/explorepy/tools.py

```diff
@@ -35,7 +35,7 @@
 logger = logging.getLogger(__name__)
 lock = Lock()
 
-EXG_CHANNELS = ['ch1', 'ch2', 'ch3', 'ch4', 'ch5', 'ch6', 'ch7', 'ch8']
+EXG_CHANNELS = [f"ch{i}" for i in range(1, 33)]
```

Comment:
32 should be a variable that can be accessed from multiple places (and changed if more channels are added) (so instead this should read `range(1, max_channels+1)`